src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -12,8 +13,11 @@
 import com.minecolonies.structures.fake.FakeWorld;
 import com.minecolonies.structures.lib.ModelHolder;
 import net.minecraft.block.Block;
+import net.minecraft.block.BlockBeacon;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.block.BlockBeacon'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)